XenAdmin/Wizards/NewNetworkWizard.cs

```diff
@@ -211,9 +211,12 @@ private void CreateSRIOV()
             if (pool == null)
                 return;
 
+            if (pageSriovDetails.SelectedHostNic == null)

```

Comment:
In addition to this, we should also disable the Finish button on NetWSriovDetails when there is nothing selected. This can be done by overriding EnableNext()